docs/book/helpers/intro.md

```diff
@@ -77,7 +78,7 @@ for, and rendering, the various HTML `<head>` tags, such as `HeadTitle`,
 - [Placeholder](placeholder.md)
 - [Url](url.md)
 
-> ### Help document!
+> ### Help us document the helpers
```

Comment:
```suggestion
> ### Help Us Document the Helpers
```

For headlines the Chicago Manual of Style capitalization rules must be used:

1. Capitalize the first and the last word.
1. Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
1. Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
1. Lowercase the ‘to’ in an infinitive (I want to play guitar).

The online tool [capitalizemytitle.com](https://capitalizemytitle.com/style/Chicago/) can help to capitalize and case convert the headlines.